src/screens/controllers/UScreenSingController.pas

```diff
@@ -1088,9 +1088,9 @@ procedure TScreenSingController.LoadNextSong();
   begin
     fVideoClip := VideoPlayback.Open(VideoFile);
     fCurrentVideo := fVideoClip;
```

Comment:
would it be possible to also move this line to inside the `if`? the function initially sets `fCurrentVideo := nil` at the beginning, it would make things more intuitive.
technically the PR as-is would fix the problem it's trying to address, but the assigning _before_ the `if` is what probably threw me when I changed this. I'll leave it open for a day or so if you want to commit it yourself, otherwise I'll just do it before merging.

Reply:
done